app/src/main/java/edu/grinnell/appdev/grinnelldirectory/LoginActivity.java

```diff
@@ -2,12 +2,29 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 
 public class LoginActivity extends AppCompatActivity {
+    @BindView(R.id.username) EditText username;
+    @BindView(R.id.password) EditText password;
+    @BindView(R.id.signin) Button signin;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
+        ButterKnife.bind(this);
+    }
+
+    @OnClick(R.id.signin)
+    public void signin(View view) {
```

Comment:
Will this method be called anywhere outside this activity? I doubt that. If it isn't, this should be marked `private`. Also use camel case here as such: `public void signIn(View view)`